src/components/Card.js

```diff
@@ -1,48 +1,72 @@
     const flipCard = (event) => {
         event.preventDefault(); 
         setIsFlipped(!isFlipped); 
     }
-
-    // Now we must connect the ReactFlipCard component to the isFlipped State, so that it will know at any time what the state value is 
-    // flipDirection is a react-flip-card library property that specifies how the card flips
+    //* isFlipped is connected to the ReactCardFlip component - the parents of two separate containing divs - the front and the back.
+    //* front and back of each each is its own Card component - the sides have slightly different functionality
+    //* each "face" of the card is connected to the flipCard handler fn
     return (
-        <Card className={classes.root}>
+        <React.Fragment>
+        
             <ReactCardFlip isFlipped={isFlipped} flipDirection="vertical">
-                {/* 2 main DIVS will go inside the parent ReactCardFlip component - they are the front and back respectively */}
-                {/*//* can I use MATERIAL UI with this library? Try replacing divs in a moment, perhaps it can be 2 child elements*/}
-                <CardContent onClick={flipCard}>
-                        <Typography variant="h4" component="p">{card.card_front}</Typography>
-                </CardContent>
-
-                <CardContent onClick={flipCard}>
-                        <Typography variant="h5" component="p">{card.card_back}</Typography>
-                </CardContent>
+                <Card onClick={flipCard} className={classes.root} variant="outlined">
+                    <CardHeader
+                        avatar={<Avatar className={classes.small} aria-label="subject">CSS</Avatar>}
+                        className={classes.header}
+                        title="Deck Subject Here"
+                    />
+                    <CardContent className={classes.content}>
+                        <Typography className={classes.text} variant="h4">{card.card_front}</Typography>
+                    </CardContent>
+                    <CardActions>
+                        <Button size="small" variant="outlined" color="secondary">Delete</Button>
+                        <Button size="small" variant="outlined">Edit</Button>
+                    </CardActions>
+                </Card>
+
+                <Card onClick={flipCard}>
+                    <CardContent className={classes.content}>
+                        <Typography className={classes.text} variant="h4">{card.card_back}</Typography>
+                    </CardContent>
+                    <CardActions>
+                        <Button size="small" variant="outlined">Edit</Button>
+                    </CardActions>
+                </Card>
             </ReactCardFlip>
-        </Card>
+
+        </React.Fragment>
     ); 
 }
 
 export default withStyles(styles)(FlashCard); 
 
-// ! Problem - when the cards are flipped (clicked) they change width, and cause their parent to collapse - it's an ugly flick in the UI 
```

Comment:
This was a stupid easy fix. I was just nesting an unnecessary level of divs deep.